Automattic/wp-calypso
# AI Logo Generator: Apply logo to site

Closes #85859

## Proposed Changes

* Implements the functionality of updating the site logo
* Links loading to the store
* Use proper buttons

## Testing Instructions



* Open the logo generator modal
* Click to save a logo to the library
* Check that the logo is saved to the library
* Generate another logo
* Use the carousel to check that the first logo is set as "saved" correctly
* Click on the button to use the logo on either logo
* Check that the "use logo" button disappears
* Refresh the page
* Check that the site's logo has changes correctly

The correct behavior would be to either reload the logo on the site or refresh the page. That is not yet done.

## Pre-merge Checklist
